docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -3501,6 +3501,89 @@ Return low-level information on the node `id`
 -   **404** – no such node
 -   **500** – server error
 
+### Remove a node
+
+
+`DELETE /nodes/<id>`
+
+Remove a node [`id`] from the Swarm.
```

Comment:
Hum not sure either, but the doc should reflect what the API does (and then we should fix or not the API)